news/warning.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* <news item>
+
+**Deprecated:**
+
+* Deprecation warning fixed for axes in np.fft in fourigui.py
```

Comment:
@Fil158 

Typically depreciated refers to classes and functions no longer support.  So you can put it under "Fixed" - please refer to the "Lessons from ..." GitLab guide for writing a news file.

Reply:
Ok, next time I'll do so

Reply:
No please do now. These are user facing news.